mods/voidmulti.sc2mod/base.sc2data/gamedata/abildata.xml

```diff
@@ -557,8 +598,33 @@
         <Arc value="0"/>
         <Range index="0" value="5"/>
     </CAbilEffectTarget>
+    <CAbilMorph id="LiberatorMorphtoAA">
+        <InfoArray index="0">
+            <SectionArray index="Abils">
+                <DurationArray index="Delay" value="0"/>
```

Comment:
Fixed an issue where Liberators could not be given certain orders immediately after being ordered to unsiege

For orders, the abilities need to be swapped immediately. This should allow queuing of commands like attacking an air, sieging at another position and maybe even moving.
I did not test what exactly was not possible before.